tests/mocks.cpp

```diff
@@ -124,6 +125,17 @@ void setup_mock_bpftrace(MockBPFtrace &bpftrace)
   bpftrace.structs.Lookup("struct _tracepoint_file_filename")
       .lock()
       ->AddField("filename", ptr_type, 8, std::nullopt, false);
+  bpftrace.structs.Add("struct _tracepoint_btf_tag", 8);
+  auto ptr_type_w_tag = CreatePointer(CreateInt8());
+  ptr_type_w_tag.SetBtfTypeTags({ "rcu" });
+  auto ptr_type_w_bad_tag = CreatePointer(CreateInt8());
+  ptr_type_w_bad_tag.SetBtfTypeTags({ "rcu", "percpu" });
+  bpftrace.structs.Lookup("struct _tracepoint_btf_tag")
+      .lock()
+      ->AddField("parent", ptr_type_w_tag, 8, std::nullopt, false);
```

Comment:
```suggestion
      ->AddField("parent", ptr_type_w_tag, 0, std::nullopt, false);
```

Reply:
These are supposed to be 8 and 16 - though TBH, I don't quite understand why. The only field marked as offset 0 above is one that fails the unit test e.g. `test("tracepoint:file:filename { args.common_field }", 1);`

Reply:
Oh right. The first 8 bytes of tracepoint args is some internal fields for the kernel. We don't allow the users to access those